source/extensions/common/aws/BUILD

```diff
@@ -27,6 +27,7 @@ envoy_cc_library(
         "//source/common/buffer:buffer_lib",
         "//source/common/common:hex_lib",
         "//source/common/common:logger_lib",
+        "//source/common/common:matchers_lib",
```

Comment:
Wouldn't the signer library get the `matchers_lib` from including `:utility_lib`?  Can you double check this and see if things still build and work without this line.

Reply:
nice catch! that could be removed indeed.